### PR TITLE
Added content, changed filenames, changed filepaths.

### PR DESCRIPTION
**Pull request by spacecatpixel**

### CODE REVIEW

File: docs/Mining/MiningOverview.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Mining
+
+Traditional mining is the process of extracting valuable minerals and metals from the earth. These mined minerals are scarce which creates an intrinsic value due to the resources required, the work conducted, and the time it takes to extract. 
+
+## What is Cryptocurrency Mining?
+Cryptocurrency Mining is the process of extracting Cryptocurrencies for securing and verifying transactions made on the blockchain. 
+
+Miners use computers to work on computational puzzles that change their difficulty depending on the number of miners on the network. The difficulty changes to enforce that the blocks are mined at similar time intervals, also known as the Block Generation Time. At each block generated a predetermined amount of cryptocurrency is released to the individual or party who found the solution to the puzzle. This reward is commonly known as the [Block Reward](../Advanced/Cryptoeconomics.md) and is coded to reduce its emission over time.
```
Comment (KeeJef) [inline docs/Mining/MiningOverview.md]:
```suggestion
Miners use computers to work on computational puzzles that change their difficulty depending on the number of miners on the network. The difficulty changes to enforce that the blocks are mined at similar time intervals, also known as the Block Generation Time. Each block generated produces a predetermined amount of cryptocurrency to the individual or party who found the solution to the puzzle. This reward is commonly known as the [Block Reward](../Advanced/Cryptoeconomics.md) and is coded to reduce its emission over time.
```

File: docs/Mining/MiningOverview.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Proof-Of-Work
+>> Add information about proof of work.
+
+## Mining Loki
+
+The Loki cryptocurrency can be mined through the CryptoNight Heavy hashing algorithm, which is innately [ASIC Resistant](../Mining/ASICResistance.md).
```
Comment (KeeJef) [inline docs/Mining/MiningOverview.md]:
```suggestion
The Loki cryptocurrency can be mined using the CryptoNight Heavy hashing algorithm, which is innately [ASIC Resistant](../Mining/ASICResistance.md). 
```
Remove innatley

File: docs/Mining/MiningOverview.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Proof-Of-Work
+>> Add information about proof of work.
+
+## Mining Loki
+
+The Loki cryptocurrency can be mined through the CryptoNight Heavy hashing algorithm, which is innately [ASIC Resistant](../Mining/ASICResistance.md). 
+
+### Mining Pools
+
+A mining pool is a hub where a group of miners share their processing power to the network. The rewards are split equally based on the amount of hashing power each individual adds to the block. In the early days of Bitcoin mining it was viable to mine as an individual. However, because there now exists tens of thousands of miners it has become less viable to mine individually due to the share of each individual being minute compared to the overal hash of the network.
```
Comment (KeeJef) [inline docs/Mining/MiningOverview.md]: i think this is way to general and not at all related to Loki

File: docs/ServiceNodes/ServiceNodeFunctions.md [hunk 4]
```diff
@@ -1,4 +1,22 @@
...
+
+Although the operation of exit nodes is essential to Loki’s extended utility, forcing all [Service Node](../ServiceNodes/SNOverview.md) operators to act as exit nodes could be detrimental. Acting as an exit node may expose the operator to legal risks, as users of the exit node may perform malicious activity whilst using it as a proxy. As exit nodes simply relay traffic from the internet to the end user, exit nodes often receive Digital Millennium Copyright Act (DMCA) requests or are often assumed to be the source of hacking attempts. Although in most jurisdictions safe harboring laws may protect exit node operators, internet service providers that carry Service Node traffic on their servers may fear legal risks and often cut off service to the exit node.
+
+Upon startup, a Service Node is assigned a relay flag and is restricted to routing packets within Lokinet, but never makes requests to the wider internet. An operator must opt-in if they wish to become an exit node, in doing so they demonstrate an understanding of the additional risks while also submitting to additional [Swarm tests](../Advanced/SwarmFlagging.md).
```
Comment (KeeJef) [inline docs/ServiceNodes/ServiceNodeFunctions.md]: This is not the case currently, this is proposed functionality

File: docs/ServiceNodes/ServiceNodeFunctions.md [hunk 5]
```diff
@@ -1,4 +1,22 @@
...
+
+Opting-in as an exit node affords an operator double the reward of a normal relay when selected for a [block reward](../Advanced/Cryptoeconomics.md). This incentive is provided to ensure that exit node operators have sufficient financial  incentives to operate exit nodes, helping to protect against [Sybil attacks](../Advanced/SybilResistance.md) specifically  targeted to take over the exit node network. This is a vulnerability which Tor suffers from due to its low ratio of exit nodes to relays.
```
Comment (KeeJef) [inline docs/ServiceNodes/ServiceNodeFunctions.md]: Also not currently the case

File: docs/Wallets/WalletsOverview.md [hunk 6]
```diff
@@ -4,25 +4,45 @@ The Loki wallet is a gateway to private decentralised transactions and communica
 
 >> TO DO: What do Wallets do?
 
-#### Web Wallet (Loki Locker)
-The quickest way to access a Loki Wallet is through a web based wallet, [Loki Locker](https://lokilocker.com/). The [Web Wallet](../Wallets/WebWallet/WebWalletOverview.md) is the easiest to use and doesn't require any downloaded files.
+## Web Wallet (Loki Locker)
+The quickest way to access a Loki Wallet is through a web based wallet, [Loki Locker](https://lokilocker.com/). The Web Wallet is the easiest to use and doesn't require any downloaded files.
```
Comment (KeeJef) [inline docs/Wallets/WalletsOverview.md]: this should not be recommended until it works

File: docs/index.md [hunk 7]
```diff
@@ -6,24 +6,22 @@ Privacy is having the agency to decide when you reveal personal information. It
 
 Loki provides the means to transact and communicate privately and anonymously, using the decentralised nature of blockchain technology to create new methods of interacting with the internet.
 
-Loki is a private cryptocurrency based on Monero. Over the course of the coming months, the Loki project aims to offer an incenvised full node layer with a secondary p2p network that offers a private communications layer based on the Tor protocol.
+Loki is a private cryptocurrency based on Monero. Over the course of the coming months, the Loki project aims to offer an incenvised full node layer with a secondary peer-to-peer network that offers a private communications layer based on the Tor protocol.
```
Comment (KeeJef) [inline docs/index.md]: perhaps refer to this https://github.com/loki-project/loki/pull/298

File: docs/Wallets/WalletsOverview.md (hunk 6 above)
Comment (Sonofotis) [reply docs/Wallets/WalletsOverview.md]: I was unaware that the loki locker wasnt working

File: docs/Mining/MiningOverview.md (hunk 3 above)
Comment (Sonofotis) [reply docs/Mining/MiningOverview.md]: Mining Pools are related to Loki. When individuals go into the docs they should know that it's better to mine in a pool rather then mine out of the GUI.

File: docs/Advanced/CryptoNoteElements.md [hunk 8]
```diff
@@ -1,15 +1,14 @@
 # CryptoNote Elements
 
-Although a full-node incentives scheme could be implemented on top of any cryptocurrency, Loki uses the Monero source code because of the high level of privacy it affords to transactions. Monero is an evolution on the [CryptoNote protocol](https://cryptonote.org/whitepaper.pdf), which uses [ring signatures](#ring-signatures), [stealth addresses](#stealth-addresses), and [RingCT](#ringct), giving users the ability to sign transactions and obfuscate amounts while maintaining plausible deniability.
+CryptoNote is an [abstraction layer](https://en.wikipedia.org/wiki/Abstraction_layer) that specifies the shared communications protocols and interface methods used by nodes in a communications network. The CryptoNote protocol powers several decentralised privacy focused digital currencies.
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/Advanced/CryptoNoteElements.md]: Removed the first paragraph

File: docs/Advanced/Cryptoeconomics.md [hunk 9]
```diff
@@ -1,22 +1,49 @@
...
-and write transactions into blocks and collect fees for doing so.  As a consensus rule in Loki,
-each block contains multiple reward outputs of which only one goes to the miner.
+## Block Rewards
 
-As a result of this study, on the 30th of July, 2018, at block 64324, the Loki block reward went from being calculated in terms of the circulating supply with an emission speed factor of 20(link to Monero stuff), to be derived from the block height. The formula used to calculate the block reward `BR` where `h` is the block height is:
+A block reward is the reward created after a new block has been generated. The reward is then distributed to the peers in the network that helped facilitate the generation of the block. In most cases, the [miners](../Mining/MiningOverview.md) are the individuals who receive this reward as they collect and write transactions into blocks. 
+
+[Proof-of-work](/Mining/MiningOverview/#proof-of-work) is used to ensure only 1 miner wins the right to embed transactions on the blockchain and thus only 1 miner will receive the reward.
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/Advanced/Cryptoeconomics.md]: Removed due to your reasoning. POW does not need to be in the cryptoeconomics section.

File: docs/Advanced/Cryptoeconomics.md [hunk 10]
```diff
@@ -1,22 +1,49 @@
...
+
+The amount of Loki rewarded `BR` to the network at each block height `h` follows the following equation:
 
 <img src="https://latex.codecogs.com/svg.latex?\Large&space;BR = 28 + \frac{100}{2^{(\frac{h}{64800})}}" title="\Large BR = 28 + \frac{100}{2^{\frac{*h*}{64800}}}" />
 
+This block reward formula is a result of [Loki's Cryptoeconomic study](https://loki.network/wp-content/uploads/2018/08/LokiCryptoeconomicHistoric.pdf), on the 30th of July, 2018, at block 64324, the Loki block reward went from being calculated in terms of the circulating supply with an emission speed factor of 20, to be derived from the block height.
+
+### Block Reward Split
+
+As Service Node's went live on 20th of September, 2018, at block 101250, the addition of a Block Reward split was required to financially incentive the [Service Node](../ServiceNodes/SNOverview.md) Operators to maintain and run a Service Node. During the fork on the 30th of July, 2018, the Block Reward was hard coded to split the reward as follows:
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/Advanced/Cryptoeconomics.md]: Applied

File: docs/Advanced/Cryptoeconomics.md [hunk 11]
```diff
@@ -1,22 +1,49 @@
...
+
+#### Service Node Reward
+The second output in each block (50% of total reward) goes to a [Service Node](../ServiceNodes/SNOverview.md), or two Service Nodes if a relay is selected. Service Nodes are rewarded based on the time since they last received a reward (or time since they registered), with a preference for nodes that have been waiting longer. Each time a Service Node registers with the network it assumes the last position in the queue. If the Service Node maintains good service and is not ejected from the queue by a [swarm flag](../Advanced/SwarmFlagging.md), it slowly migrates to the higher positions in the queue. Nodes at or near the front of the queue are eligible for a reward, and once awarded, the node again drops to the last position in the queue and begins slowly working its way back up.
+
+#### Governance Reward
+The final 5% portion of the block reward is distributed towards [governance operations](../Governance/Governance.md);  3.75% is sent to the Loki Foundations address which is derived deterministically each block and the remaining 1.25% is reserved for the outputs of a [funding block](../Governance/LokiFundingSystem.md).
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/Advanced/Cryptoeconomics.md]: Removed the loki funding block information

File: docs/index.md (hunk 7 above)
Comment (Sonofotis) [reply docs/index.md]: Changed to reflect changes from loki-project/loki#298

File: docs/Wallets/WalletsOverview.md (hunk 6 above)
Comment (Sonofotis) [reply docs/Wallets/WalletsOverview.md]: Removed content. Added to do.

File: docs/FAQ.md [hunk 12]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+
+###**Why the name Loki?**
+Loki is the Norse god of trickery. This is fitting as we use a lot of digital 'tricks' to obfuscate transaction. Also, it is a play on words as transactions and communications on the network are very 'Low-Key'.
+
+###**What is the business model behind this peer-to-peer network?**
+Loki operates on an incentives structure provided by the network. Service nodes are rewarded to operate via Proof of Service. Peer-to-peer communications only occur through the [Service Node](/ServiceNodes/SNOverview) layer which is already incentivised to run so there is a weak case for providing any additional businesses models. The business model post-launch for the Loki foundation is to continue to build both core [SNApps](/Lokinet/SNApps/) and aid in the development of 3rd party SNApps which Service nodes can optionally operate.
+
+###**What are business use cases for the project?**
+Use cases are high since Loki is building a protocol layer on top of incentivised nodes, SNApps can range from Deadman drops to Oracles and even cooler uses cases that we haven't yet thought of but open-source developers will.
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Removed what business use cases for the project are? Fleshed out the Loki operates on incentives...

[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Fleshed out, removed "what are business use cases for this project" title until the use cases have been worked out.

File: docs/FAQ.md [hunk 13]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+
+###**Will you guys commit back to the monero source code?**
+Yeah, we're planning to push back any useful changes to Monero, such as optimizations, bug fixes, and feature adds, probably an overhauled networking system as well.
+
+###**What's the difference from Monero?**
+[Fixed ring size](/Advanced/CryptoNoteElements/#ring-signature-size) with minimum mixin of 10, [governance block reward](/Advanced/Cryptoeconomics/#block-reward-split) of 5% which will [fund community projects and developments](/Governance/LokiFundingSystem), Mixin distribution changes, and [emission curve changes](/Advanced/Cryptoeconomics/#loki-block-reward).
+These are baselayer changes that will differentiate us between Monero, however the main change that is being implemented is the second layer of incentivised [Service Nodes](/ServiceNodes/SNOverview).
+
+###**What are the privacy improvements over Monero?**
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Removed FAQ completely. Monero privacy probably has not been improved by Loki. Can add again if we have.

File: docs/FAQ.md [hunk 14]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+We are statically fixing ringsizes and upping the minimum [ring size to 10](../Advanced/CryptoNoteElements/#ring-signature-size) which should make users safer against some edge case analysis, and it will improve the effectiveness of churning. were also making some slight changes to the way mixins are chosen.
+
+###**What is the difference between Loki Messenger and Mainframe messengers?**
+The main difference is that the mainframe uses an ERC 20 token to power its network and it has a pay by service model meaning you have to pay to send messages.
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Removed completely.

File: docs/FAQ.md [hunk 15]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+
+TLDR; Most messaging apps use Ethereum and will suffer scale and privacy issues because of it.
+
+###**Does Loki have smart contracts?**
+[SNApps](../LokiServices/SNApps) run off chain so they don't have a smart contract element. In general, the platform is capable to do smart contracts in the future,  but it's hard to speculate on SNApps before the protocol layer is built.
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Removed completely

File: docs/FAQ.md [hunk 16]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+###**Why is there a 5% governance reward?**
+We intended to create a [self-funding system](../Governance/LokiFundingSystem) so that users can be certain that no external influences drive the development funding of the network in an undesirable direction. The 5% block reward that is issued to the governance wallet is for this purpose.
+
+This is a similar approach to other projects, such as the Zcash Foundation, who for the first 4 years of the network’s operation will receive a 20% block reward, and the DASH project, who receive a 10% block reward from the network.
+
+We wanted to keep the amount significantly less than Zcash and DASH but want to make sure that it is enough to sustain the project indefinitely.
+
+In the future, the community may decide that this reward is unnecessary or of too high or too low a proportion, in which case, a hard fork event may change the nature of this block reward split.
+
+###**What is the block reward for mining?**
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Added link to lokiblocks.com

File: docs/FAQ.md [hunk 17]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+50% of the block reward goes to service nodes, 45% to miners, 5% to the governance pool. You can see more details on the block reward split [here](../Advanced/Cryptoeconomics/#block-reward-split)
+
+###**What is the block generation time?**
+About 120 seconds.
+
+###**Can you run multiple service nodes in a single VPS server?**
+You will need a new VPS server per [Service Node](../ServiceNodes/SNOverview).
+
+###**Where will SNApps and all of its data be hosted?**
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Not sure what extra detail to add

File: docs/FAQ.md [hunk 18]
```diff
@@ -73,3 +102,66 @@ It is not an ERC20 token. Loki is its own coin running its own mainnet. See [lok
...
+
+In the future, the community may decide that this reward is unnecessary or of too high or too low a proportion, in which case, a hard fork event may change the nature of this block reward split.
+
+###**What is the block reward for mining?**
+50% of the block reward goes to service nodes, 45% to miners, 5% to the governance pool. You can see more details on the block reward split [here](../Advanced/Cryptoeconomics/#block-reward-split)
+
+###**What is the block generation time?**
+About 120 seconds.
+
+###**Can you run multiple service nodes in a single VPS server?**
```
[the remark this answers is not in the document]
Comment (Sonofotis) [reply docs/FAQ.md]: Rewrote content.

**Pull request merged**